ENIGMAsystem/SHELL/Universal_System/dynamic_args.h

```diff
@@ -25,23 +25,26 @@
 **                                                                              **
 \********************************************************************************/
 
-#ifndef _VARARGS_H
-#define _VARARGS_H
+#ifndef ENIGMA_VARARGS_H
+#define ENIGMA_VARARGS_H
+
+#include "var4.h"
+
 namespace enigma {
-  struct varargs
-  {
-    int argc;
-    void *argv;
-    #ifndef JUST_DEFINE_IT_RUN
-    void sort();
-    void reverse();
-    variant get(int) const;
-    varargs& operator,(variant);
-    varargs();
-    varargs(variant);
-    varargs(varargs&);
-    ~varargs();
-    #endif
-  };
-}
+struct varargs {
+  int argc;
+  void* argv;
+#ifndef JUST_DEFINE_IT_RUN
```

Comment:
And, while it's now more consistent with Google style, this looked better before, honestly. No point to formatting this file.